docs/integration-testing.md

```diff
@@ -25,8 +25,8 @@ eventually we might want to write an integration test that uses a CosmosDB test
 
 ## Coding Guidelines
 
-An integration test is annotated with `@IntegrationTest`, which causes the test runner to ignore it unless the
-`RUN_INTEGRATION_TEST` environment variable is set to `true`.
+An integration test should have an annotation e.g.`@IntegrationTest, @AzureCosmosDbIntegrationTest` which causes the test runner to ignore it unless the
```

Comment:
Ok, I see now you are using these annotations to add the corresponding tags in addition to checking the `RUN_INTEGRATION_TEST` flag. Have you considered using `@IntegrationTest` together with `@Tag` directly on the tests? We can define constants for the possible tag values. I think it might lead to more concise code.

Reply:
Updated to use composite annotation and removed use of RUN_INTEGRATION_TEST environment variable. Please take a look now.